ci/indy-pool.dockerfile

```diff
@@ -33,7 +33,7 @@ ARG python3_orderedset_ver=2.0
 ARG python3_psutil_ver=5.4.3
 ARG python3_pympler_ver=0.5
 
-RUN apt-get update -y && apt-get install -y \
+RUN apt-get update -y && apt-get install -y --allow-unauthenticated \
```

Comment:
some issues with the docker build?

Reply:
https://github.com/hyperledger/indy-sdk/issues/2407